sagemaker-python-sdk/dgl_lifesci_bap/pytorch_bap.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>Branding nit: in the subsection title, capitalize <code>Estimator</code>	. There's also 2 spaces between <code>SageMaker's</code>  and <code>estimator</code>.  <span class="ql-cursor">﻿</span></p><p> </p><p>Style nit: use backticks for code formatting for parameter names, such as <code>train_instance_count</code>.</p><p> </p><p>Please note that SageMaker v2 renames parameter names, such as <code>train_instance_count</code> -&gt; <code>instance_count</code>, <code>train_instance_type</code> -&gt; <code>instance_type</code>. Ensure that you're using only v2 names: <a href="https://sagemaker.readthedocs.io/en/stable/v2.html" target="_blank">https://sagemaker.readthedocs.io/en/stable/v2.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Changed the statements